<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblDialogUI_GrayScaleImage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed Image 1 - GrayScale Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDialogUI_HistEquImage">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed Image 2 - Histogram Equalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDialogUI_HSV">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed Image 3 - HSV Color Model</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
